src/pydata_sphinx_theme/assets/styles/extensions/_sphinx_design.scss

```diff
@@ -322,10 +322,12 @@ details.sd-dropdown {
       top: 0.7rem;
     }
 
+    @include hover-darken-lighten;
+
     // Focus ring
-    &:focus-visible {
+    &:focus:focus-visible {
```

Comment:
This reimplements https://github.com/pydata/pydata-sphinx-theme/pull/2060. But I made that PR separate because it is a bug fix.